app/aws/PasskeyDB.scala

```diff
@@ -235,4 +235,44 @@ object PasskeyDB {
       )
     )
   )
+
+  def loadCredentialByName(
+      user: UserIdentity,
+      passkeyName: String
+  )(implicit dynamoDB: DynamoDbClient): Try[QueryResponse] =
+    Try {
+      val expressionValues = Map(
+        ":username" -> AttributeValue.fromS(user.username),
+        ":passkeyName" -> AttributeValue.fromS(passkeyName)
+      )
+      val request = QueryRequest
+        .builder()
+        .tableName(tableName)
+        .keyConditionExpression("username = :username")
+        .filterExpression("passkeyName = :passkeyName")
+        .expressionAttributeValues(expressionValues.asJava)
+        .build()
+      dynamoDB.query(request)
+    }.recoverWith(err =>
+      Failure(JanusException.failedToLoadDbItem(user, tableName, err))
+    )
```

Comment:
This doesn't appear to be used anywhere
```suggestion
```

Reply:
Done in https://github.com/guardian/janus-app/pull/571/commits/1ed5f52fa8421cd25f92e47d967e8f63a4f17b5e